clients/cli/src/main/java/org/apache/gravitino/cli/commands/ListGroups.java

```diff
@@ -54,9 +58,29 @@ public void handle() {
     }
 
     if (groups.length == 0) {
-      printInformation("No groups found in metalake " + metalake);
+      printInformation("No groups found in metalake: " + metalake);
```

Comment:
Please do not change the format of existing messages unless it is actually needed